src/main/resources/META-INF/plugin.xml

```diff
@@ -17,7 +17,7 @@
     </change-notes>
 
     <depends>com.intellij.modules.java</depends>
-    <depends optional="true" config-file="plugin.xml">org.intellij.scala</depends>
+    <depends optional="true" config-file="">org.intellij.scala</depends>
```

Comment:
Was that `config-file` attribute unnecessary in the first place or does this change something?

Reply:
surprisingly it does... with the attribute non-empty, the `verifyPlugin` task of Gradle IJ plugin failed with stackOverFlow error, whereas this way it works (tests pass, `runIde` starts, _etc._). It's a bit weird that this appeared only when I merged to `test`, however, this change solves the issue.

Reply:
magic...